src/proc/split-filter.cpp

```diff
@@ -0,0 +1,184 @@
+// License: Apache 2.0. See LICENSE file in root directory.
+// Copyright(c) 2020 Intel Corporation. All Rights Reserved.
+
+#include "split-filter.h"
+
+namespace librealsense
+{
+    split_filter::split_filter()
+        : generic_processing_block("Split Filter"),
+        _selected_stream_id(1.f)
+    {
+        auto selected_stream_id = std::make_shared<ptr_option<float>>(0.f, 2.f, 1.f, 1.f, 
+            &_selected_stream_id, "Selected stream id for display",
+            std::map<float, std::string>{ {0.f, "0"}, { 1.f, "1" }, { 2.f, "2" }});
```

Comment:
Maybe use `origin` instead of `0` to make it easier for the users?

Reply:
Done